src/components/application_manager/rpc_plugins/rc_rpc_plugin/src/rc_app_extension.cc

```diff
@@ -88,7 +88,8 @@ void RCAppExtension::UnsubscribeFromInteriorVehicleData(
 void RCAppExtension::UnsubscribeFromInteriorVehicleDataOfType(
     const std::string& module_type) {
   bool unsubscribed = false;
-  for (auto& item : subscribed_interior_vehicle_data_) {
+  auto subscribed_ivi = subscribed_interior_vehicle_data_;
+  for (auto& item : subscribed_ivi) {
```

Comment:
```suggestion
  for (const auto& item : subscribed_ivi) {
```

Reply:
fixed in e6e5767ef10ee27415d93deeac04dd30739b2f6e